docs/root/intro/version_history.rst

```diff
@@ -4,6 +4,7 @@ Version history
 1.11.0 (Pending)
 ================
 * upstream: added :ref:`upstream_cx_pool_overflow <config_cluster_manager_cluster_stats>` for the connection pool circuit breaker.
+* redis: add support for zpopmin and zpopmax commands.
```

Comment:
Sorry just missed this. Can you alpha sort? Sorry and than you.

Reply:
Sorry I actually meant alpha sort this entry in the version history, but thanks for the other one also. :)

Might as well merge master again also.

/wait